### DISCUSSION TITLE
Unlimited archive experience

### DISCUSSION THREAD

**Discussion by azenMatt** [community/community#52805] [Projects and Issues] [answered]
As part of our journey to increase the number of items that can be added to a project from 1200, as per this [feedback](https://github.com/orgs/community/discussions/9678) we have now released an alpha of an 'unlimited archive' experience. 

This discussions is to: 

1. Collect feedback on this experience 
2. A place where you can request to be added to the alpha (please let me know the org name and project number) 

If you are in the alpha and did not request to be added, this is because your project was identified as one with a large archive which could benefit from this new experience. If you wish to revert to the old experience please comment here and we can do so (you will need to first make sure your project has an archive less that 10k items) 

The unlimited archive will eventually be the same as the current experience (just with more items), however while we iterate towards this today it has the limitations that you can only search over the archive with `is:`, `assignee:` and custom single selects. We will be adding more filter options on a regular basis. 

![image](https://user-images.githubusercontent.com/7584089/232046772-cda6762c-e302-42d7-b5e0-609c6bdf6d2a.png)

Please let us know what you think and we will keep you updated as we continue to progress towards an unlimited projects experience!

**Comment by tnykodym** [comment]
Hi @azenMatt , can you please add my project (under the enterprise org) into the alpha? https://github.com/orgs/kyndryl-global-delivery/projects/1
Thank you

**Reply by azenMatt** [reply]
Done!

**Reply by tnykodym** [reply]
thank you @azenMatt , working well for us, without any issues.
Will the limit be adjusted also for the active project items?

**Reply by azenMatt** [reply]
Yes - this is incrementally in that direction and we are actively working on that increase at the moment!

**Comment by maxisam** [comment]
@azenMatt me too, thanks! https://github.com/orgs/Millers-IT/projects/5

**Reply by azenMatt** [reply]
Added :) let me know how it goes!

**Comment by ss186222** [comment]
is there a rough/projected timeline as to when the issue limit will be increased? Thanks!

**Reply by azenMatt** [reply]
Not right now. But this calendar year as it is in active development

**Reply by azenMatt** [reply]
As we get closer we can share a more accurate date

**Reply by og250049** [reply]
Great news. Do you know if Project issues will continue to have some limitation or if it will be unlimited like beta archived issues?

**Reply by azenMatt** [reply]
it's likely to be a high limits, we need to run more tests as we get closer to releasing - but are thinking in the region of 50k

Would that be enough for your use-case?

**Reply by og250049** [reply]
That will be enough for our use case, yes. With the current 1,200 issue limitation, we are currently investigating possible solutions until the 50k issue is rolled out. Would it be possible to enroll in the Alpha to have early access to this feature?

**Reply by azenMatt** [reply]
Yes - as soon as we are ready for a beta I'l post back here 😄

**Reply by og250049** [reply]
@azenMatt is there any rough target timeline for when users should expect this functionality to be available?

**Reply by og250049** [reply]
@azenMatt can you please add our enterprise project under the unlimited archive alpha? Thank you!

https://github.com/orgs/ncr-swt-retail/projects/12/

**Reply by azenMatt** [reply]
Added!

**Reply by vm250200** [reply]
@azenMatt Do we have a timeline for the rollout to be available ?

**Reply by azenMatt** [reply]
Mid-to-late Jan for the Beta

**Reply by vm250200** [reply]
Thanks for the update.

**Reply by vm250200** [reply]
@azenMatt Happy New Year. As we are in 2024 now, do we have any concrete date for the rollout ?

**Comment by Bexolo** [comment]
@azenMatt Please can we be added to this alpha? https://github.com/orgs/OurPeople/projects (Project 2) Would be happy to provide feedback. Thank you!

**Reply by azenMatt** [reply]
Added 😄

**Reply by Bexolo** [reply]
Thank you!

**Reply by Bexolo** [reply]
@azenMatt - Hi again. We are seeing the 'maximum number of items reached' error despite being added to this alpha. Please could you double check we are definitely on it? If we are definitely on it, what info do you need from us to help diagnose the issue? Thanks

**Reply by azenMatt** [reply]
That error is in the archive or the main project?

**Reply by Bexolo** [reply]
@azenMatt The error occurred when creating a new issue within one of the repos.

**Reply by Bexolo** [reply]
<img width="1669" alt="Screenshot 2023-09-13 at 6 03 11 pm" src="https://github.com/community/community/assets/31006885/2bbe7c20-ccfa-4087-99e6-223025de52aa">
Screenshot for your reference.

**Reply by azenMatt** [reply]
This is because the alpha so far only allows unlimited items in the archive of a project. We are working on bringing it to the main project and are looking that have this ready by the end of the year

**Comment by unicaf-devs** [comment]
Hi @azenMatt We have also hit that limit, however in our case we are on the Enterprise Server versioned 3.10.

Is there a way for us to participate in this alpha.

Is there any plans on including this into next versions

My best regards

**Reply by azenMatt** [reply]
Hi, this won't be available on GHES until we move it out of beta into GA. But we will keep pushing to make that happen as soon as possible

**Comment by dsdexcom** [comment]
Hi @azenMatt Can we please also be added to this alpha? https://github.com/orgs/dexcom-inc/projects/20

We're happy to provide feedback and help however we can. Thanks!

**Reply by azenMatt** [reply]
Done!

**Comment by dinhtrungndt** [comment]
ohh #1

**Comment by jeromelaban** [comment]
Hi @azenMatt, can we please be added into the alpha? https://github.com/orgs/unoplatform/projects/63

Thank you very much!

**Reply by azenMatt** [reply]
Done :)

**Comment by arapov** [comment]
Hi @azenMatt, we'd like to be added to the alpha too: https://github.com/orgs/openssl/projects/2.

**Reply by azenMatt** [reply]
Added!

**Comment by darpank** [comment]
Hi @azenMatt, if possible we'd like to be added as well to the alpha: https://github.com/orgs/trueanthem/projects/22. Thanks so much, DK.

**Reply by azenMatt** [reply]
Added!

**Comment by nkolev92** [comment]
Hello! 
@azenMatt Can https://github.com/orgs/NuGet/projects/13 be added to this alpha?

Thanks!

**Comment by primeroz** [comment]
@azenMatt 

```
The unlimited archive will eventually be the same as the current experience (just with more items), however while we iterate towards this today it has the limitations that you can only search over the archive with is:, assignee: and custom single selects. We will be adding more filter options on a regular basis.

```

is this search limitation only for the archive or also for the project backed by the unlimited archive feature ?

**Comment by ss186222** [comment]
Hi @azenMatt, do we have any potential timelines for the rollout? Is this expected to be released sometime this year?

**Comment by evi-liu** [comment, marked as answer]
👋 Hi all, as we mentioned, the unlimited archive experience was a milestone on our journey to enabling unlimited items. I'm happy to share that the private beta for unlimited project items has shipped! 🎉 

If you're interested, please read the [changelog](https://github.blog/changelog/2024-02-12-github-issues-projects-projects-without-limits-private-beta/) for more details, including how to join the waitlist and what is currently supported versus still not yet supported. 

If you are a project admin and you're not seeing the `Join waitlist` banner, please see this [Community Discussion](https://github.com/orgs/community/discussions/106785) for more details on how you can opt in to the private beta.

**Reply by Loi22789** [reply]
Méo hiểu gì hệt

**Comment by smith** [comment]
On my team we archive all items at the end of the week, then in our Monday morning meeting we open the archive and go through the items we completed.

With unlimited archive available it's only showing me 37 items that are all very old, but I would like to see the items I have most recently archived.
